bengans_biluthyrning_test.py::test_existens::test_start
--------------------------------------------------------------------
Last Page: https://ntig.github.bengans%20whatever/
  Browser: chrome (104.0.5112.102)
   Driver: chromedriver (104.0.5112.79)
Timestamp: 1661950542  (Unix Timestamp)
     Date: Wednesday, 31 August 2022
     Time: 2:55:42 PM  (W. Europe Daylight Time, UTC+01:00)
--------------------------------------------------------------------
Traceback: File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1776.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 591, in run
    self._callTestMethod(testMethod)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1776.0_x64__qbz5n2kfra8p0\lib\unittest\case.py", line 549, in _callTestMethod
    method()
  File "C:\Users\Nikole\Documents\GitHub\Bengans-Biluthyrning\tests\bengans_biluthyrning_test.py", line 8, in test_start
    self.open(startPage)
  File "C:\Users\Nikole\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\seleniumbase\fixtures\base_case.py", line 232, in open
    raise Exception(e.msg)
Exception: unknown error: net::ERR_NAME_NOT_RESOLVED
  (Session info: chrome=104.0.5112.102)